app/controllers/boards_controller.rb

```diff
@@ -0,0 +1,5 @@
+class BoardsController < ApplicationController
```

Comment:
Is there supposed to be a `create` method here? You have a create test as well as a create route but no create method inside the controller.

Reply:
Yes, https://online.bootcampspot.com/tdd/4

Reply:
I am not doing the create. I'm just producing the board.